row_050_2/row_050_2_unsafe.py

```
@@ -0,0 +1,344 @@
+        stu = name.get()
+        re1 = regno.get()
+        at = att.get()
+        connection = pymysql.connect(host='localhost', user='root', password='', db='class')
+        cursor = connection.cursor()
+        q = ("insert into "+str(classname)+" values(%s,%s,%s)")
+        values=[stu,re1,at]
+        try:
+            cursor.execute(q,values)
+            connection.commit()
+            st.config(text="Successful",bg="GREEN",fg="Dark GREEN")
+        except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)